File: HFSGuardaDiretorioVS2022/HFSGuardaDiretorio_CSharp/comum/DiretorioXML.cs
/*
 * Created by SharpDevelop.
 * User: henrique.souza
 * Date: 03/07/2015
 * Time: 14:53
 * 
 * To change this template use Tools | Options | Coding | Edit Standard Headers.
 */
using System;
using HFSGuardaDiretorio.objetos; 

namespace HFSGuardaDiretorio.comum
{
	/// <summary>
	/// Description of DiretorioXML.
	/// </summary>
	public class DiretorioXML
	{
	    private Diretorio diretorio;	    

		public DiretorioXML()
		{
	        this.diretorio = new Diretorio();
		}
		
		public Diretorio Diretorio {
			get { return diretorio; }
			set { diretorio = value; }
		}
		
	}
}

File: HFSGuardaDiretorioVS2022/HFSGuardaDiretorio_VBNet/comum/DiretorioXML.vb
'
' * Created by SharpDevelop.
' * User: henrique.souza
' * Date: 03/07/2015
' * Time: 14:53
' * 
' * To change this template use Tools | Options | Coding | Edit Standard Headers.
' 

Imports HFSGuardaDiretorio.objetos

Namespace comum
	''' <summary>
	''' Description of DiretorioXML.
	''' </summary>
	Public Class DiretorioXML
		Private m_diretorio As Diretorio

		Public Sub New()
			Me.m_diretorio = New Diretorio()
		End Sub

		Public Property Diretorio() As Diretorio
			Get
				Return m_diretorio
			End Get
			Set
				m_diretorio = value
			End Set
		End Property

	End Class
End Namespace
